pdns/dnsdistdist/dnsdist-kvs.cc

```diff
@@ -96,8 +96,8 @@ bool LMDBKVStore::getValue(const std::string& key, std::string& value)
       return false;
     }
   }
-  catch(const std::exception& e) {
-    warnlog("Error while looking up key '%s' from LMDB file '%s', database '%s': %s", key, d_fname, d_dbName, e.what());
+  catch (const std::exception& e) {
+    vinfolog("Error while looking up key '%s' from LMDB file '%s', database '%s': %s", key, d_fname, d_dbName, e.what());
```

Comment:
I wonder if "info" is too low here, as "not found" is not an error condition

Reply:
I agree `info` is a bit low, but this is not something we might want to be logging for every query either if something goes wrong, which is why I have gated it behind the `verbose` flag. But perhaps it means that we need `verrlog` and `vwarnlog` (this looks ugly, but you get the idea) so that we can log at a meaningful level in verbose mode?

Reply:
Agreed, don't want this repeated 1000 times per second. New `v*` calls make sense to me!

Reply:
(merging this as info for now also makes sense to me)

Reply:
I added a comment in https://github.com/PowerDNS/pdns/pull/13156#issuecomment-1740585643 since I'm refactoring a fair amount of the logging code in that PR anyway.

Reply:
ah, makes sense!